File: core/CreateTournament/Models/Player.cs
using System.Collections.ObjectModel;
using System.ComponentModel.DataAnnotations.Schema;

namespace CreateTournament.Models
{
    public class Player: DeleteableEntity
    {
        public string Name { get; set; }
        public string? ImagePlayer { get; set; }
        public int TeamId { get; set; }
        [ForeignKey("TeamId")]
        public Team Team { get; set; }
        public Collection<PlayerStats> PlayerStats { get; set; }

    }
}

File: client/src/app/core/models/classes/Player.ts
import { IPlayer } from "../interfaces/IPlayer";

export class Player implements IPlayer {
	id: number;
	name: string;
	imagePlayer?: string | undefined;
	teamId: number;

	constructor() {
		this.id = 0;
		this.name = "";
		this.imagePlayer = "";
		this.teamId = 0;
	}
}
